### PR TITLE
[countries] Resolving conflicts

### LINKED ISSUE #163: US flag is set to unknown.png while there is a US flag in the assets

### LINKED ISSUE #169: Lat, Long wrong for some countries
In, https://corona.lmao.ninja/countries?sort=%7Bparameter%7D, end-point, the USA's and several other countries lat and long are set to {0,0}. Can you please fix them?

By the way, thank you so much for a simple API with variety of variables it helps a lot for my project.

### LINKED ISSUE #172: Flag URL are not correct for all
USA,Iran,UK , South Korea and other countries urls are not correct.
"https://raw.githubusercontent.com/NovelCOVID/API/master/assets/flags/unknow.png"

### PR DESCRIPTION
**Pull request by buster95**
Closes #172 Closes #169 Closes #163

### CODE REVIEW

**Comment by buster95** [comment]
@ebwinters check this

File: server.js [hunk 1]
```diff
@@ -81,8 +84,25 @@ app.get("/countries/:country", async function (req, res) {
...
+  //     return req.query.strict.toLowerCase() == 'true' ?
+  //       e.country.toLowerCase() === countryData.country.toLowerCase() :
+  //       e.country.toLowerCase().includes(countryData.country)
+  //   } else {
+  //     if (countryData.country) {
+  //       return e.country.toLowerCase().includes(countryData.country.toLowerCase());
+  //     }
+  //   }
+  // });
+
```
Comment (ebwinters) [inline server.js]: just delete this comment

Comment (buster95) [reply server.js]: solved

**Comment by ebwinters** [comment]
PR is good to merge ✅ :

**Comment by ebwinters** [comment]
Closes #172 Closes #169 Closes #163

**Comment by ebwinters** [comment]
should this be a /v2/countries update?

**Comment by buster95** [comment]
> should this be a /v2/countries update?

not its just for show _id field in response, see your screen shots, _id field is missing

**Comment by ebwinters** [comment]
> > should this be a /v2/countries update?
> 
> not its just for show _id field in response, see your screen shots, _id field is missing

talking about entire PR

**Comment by buster95** [comment]
I don't think so for v2 we will introduce searches with id or isos standards

**Pull request merged**